assets/js/script.js

```diff
@@ -29,4 +31,295 @@ closeAbout.addEventListener('click', () => {
+    buttonLive: 'https://wdavidcch.github.io/portfolio/',
+    buttonSource: 'https://github.com/wdavidcch/portfolio',
+  },
+  {
+    image: './assets/img/face-project.jpg',
+    title: 'Facebook 360',
+    languages: ['Ruby on Rails', 'CSS', 'JavaScript', 'Html'],
+    description:
+      'Exploring the future of media in Facebook`s first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.',
+    buttonLive: 'https://wdavidcch.github.io/portfolio/',
+    buttonSource: 'https://github.com/wdavidcch/portfolio',
+  },
+  {
+    image: './assets/img/uber-project.jpg',
+    title: 'Uber Navigation',
+    languages: ['Ruby on Rails', 'CSS', 'JavaScript', 'Html'],
+    description:
+      'A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.',
+    buttonLive: 'https://wdavidcch.github.io/portfolio/',
+    buttonSource: 'https://github.com/wdavidcch/portfolio',
+  },
+  {
+    image: './assets/img/laptop-project.jpg',
+    title: 'Profesional Art Printing Data More',
+    languages: ['Ruby on Rails', 'CSS', 'JavaScript', 'Html'],
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry`s standard',
+    buttonLive: 'https://wdavidcch.github.io/portfolio/',
+    buttonSource: 'https://github.com/wdavidcch/portfolio',
+  },
+];
+
+// projects dynamically appears on the page
+
+const cardsContainer = document.querySelector('.cards-container');
+cardsContainer.innerHTML = `<article class="product-card">
+<img src="./assets/img/projects.jpg" alt="img" />
+<div class="product-info">
+    <h2>Keeping track of hundreds of components</h2>
+    <div>
+        <ul class="leng">
+            <li>Ruby on Rails</li>
+            <li>Css</li>
+            <li>JavaScript</li>
+            <li>Html</li>
+        </ul>
+    </div>
+    <button type="button" class="project-button" value="See">See Project</button>
+</div>
+</article>
+
+<article class="product-card">
+<img src="./assets/img/medicalproject.jpg" alt="img" />
+<div class="product-info">
+    <h2>Medical Illustration Sets</h2>
+    <div>
+        <ul class="leng">
+            <li>Ruby on Rails</li>
+            <li>Css</li>
+            <li>JavaScript</li>
+            <li>Html</li>
+        </ul>
+    </div>
+    <button type="button" class="project-button2" value="See">See Project</button>
+</div>
+</article>
+
+<article class="product-card">
+<img src="./assets/img/girlproject.jpg" alt="img" />
+<div class="product-info">
+    <h2>Multi Posts Stories</h2>
+    <div>
+        <ul class="leng">
+            <li>Ruby on Rails</li>
+            <li>Css</li>
+            <li>JavaScript</li>
+            <li>Html</li>
+        </ul>
+    </div>
+    <button type="button" class="project-button3" value="See">See Project</button>
+</div>
```

Comment:
Please fix the `desktop-pop` window `See Live` and `See Source` currently they are floating on top of the texts. Consider giving `max-width` for your left side text and `ul`s and make your right side buttons a little up. That way they will be side by side as the design provided.

| Your Design | Figma Design |
|-------------|---------------|
| <img width="639" alt="image" src="https://user-images.githubusercontent.com/34578631/174143681-924ac410-7e03-4a6a-aa36-97130e71e5a4.png"> | <img width="807" alt="image" src="https://user-images.githubusercontent.com/34578631/174143740-9d70222a-f3bb-451b-8966-60ba1a9551ad.png"> |

Reply:
Got it, thank you, that happened because I use position absolute for the buttons div. I will fix that using grid.